base/database/testing.go

```diff
@@ -2,11 +2,91 @@ package database
 
 import (
 	"app/base/models"
+	"app/base/utils"
 	"github.com/stretchr/testify/assert"
 	"testing"
 	"time"
 )
 
+func DebugWithCachesCheck(part string, fun func()) {
```

Comment:
This is not used anywhere. It would be useful to allow enabling this checking after evaluation using env var (ENABLE_CACHES_CHECK, default false). But we could reduce checking on single account only.

Reply:
I added the method for tracking down bugs and kept it for test purposes. I don't want to run it in production environments, since it does not provide enough information.

Invalid caches appeared due to concurrency issues, and method was pretty much useless for tracking down where the issue was.